MonoGame.Framework/Graphics/Model.cs

```diff
@@ -1,44 +1,49 @@
-﻿using Microsoft.Xna.Framework;
-using Microsoft.Xna.Framework.Content;
-using System;
-using System.Diagnostics;
+﻿using System;
 using System.Collections.Generic;
-using System.Collections.ObjectModel;
 
 namespace Microsoft.Xna.Framework.Graphics
 {
-	public class Model
+    /// <summary>
+    /// A basic 3D model with per mesh parent bones.
+    /// </summary>
+	public sealed class Model
 	{
 		private static Matrix[] sharedDrawBoneMatrices;
 		
 		private GraphicsDevice graphicsDevice;
-        private GraphicsDevice GraphicsDevice { get { return this.graphicsDevice; } }
-
-		// Summary:
-		//     Gets a collection of ModelBone objects which describe how each mesh in the
-		//     Meshes collection for this model relates to its parent mesh.
-		public ModelBoneCollection Bones { get; private set; }
-		//
-		// Summary:
-		//     Gets a collection of ModelMesh objects which compose the model. Each ModelMesh
-		//     in a model may be moved independently and may be composed of multiple materials
-		//     identified as ModelMeshPart objects.
-		public ModelMeshCollection Meshes { get; private set; }
-		//
-		// Summary:
-		//     Gets the root bone for this model.
-		public ModelBone Root { get; set; }
-		//
-		// Summary:
-		//     Gets or sets an object identifying this model.
-		public object Tag { get; set; }
-
-		public Model()
+
+        /// <summary>
+        /// A collection of <see cref="ModelBone"/> objects which describe how each mesh in the
+        /// mesh collection for this model relates to its parent mesh.
+        /// </summary>
+        public ModelBoneCollection Bones { get; private set; }
+
+        /// <summary>
+        /// A collection of <see cref="ModelMesh"/> objects which compose the model. Each <see cref="ModelMesh"/>
+        /// in a model may be moved independently and may be composed of multiple materials
+        /// identified as <see cref="ModelMeshPart"/> objects.
+        /// </summary>
+        public ModelMeshCollection Meshes { get; private set; }
+
+        /// <summary>
+        /// Root bone for this model.
+        /// </summary>
+        public ModelBone Root { get; set; }
+
+        /// <summary>
+        /// Custom attached object.
+        /// <remarks>
+        /// Skinning data is example of attached object for model.
+        /// </remarks>
+        /// </summary>
+        public object Tag { get; set; }
+
+		internal Model()
 		{
 
 		}
 
-		public Model(GraphicsDevice graphicsDevice, List<ModelBone> bones, List<ModelMesh> meshes)
+        internal Model(GraphicsDevice graphicsDevice, List<ModelBone> bones, List<ModelMesh> meshes)

```

Comment:
In XNA there is no public constructor. I really doubt that someone needs to construct it manually from parts. These constuctors in general used only by content pipeline.